src/main/resources/graphql/category.graphqls

```diff
@@ -0,0 +1,9 @@
+type Category{
+    id:ID!
+    name:String
+}
+
+extend type Query {
+    allCategories: [Category]
+    categoryById(id: ID): Category
```

Comment:
@denomelchenko Is `id` nullable? Is its return type (`Category`) nullable? The same applies to any other `Query` you have.